ibis/backends/risingwave/__init__.py

```diff
@@ -586,3 +586,9 @@ def drop_sink(
         )
         with self._safe_raw_sql(src):
             pass
+
+    @property
+    def _session_temp_db(self) -> str | None:
```

Comment:
This was actually enough to make tests pass, since otherwise `get_schema` works just fine.